package.json

```diff
@@ -1,77 +1,62 @@
 {
-  "name": "mjml-react",
-  "version": "2.0.2",
+  "name": "@faire/mjml-react",
+  "version": "2.1.0",
   "license": "MIT",
-  "author": {
-    "name": "Mantas Miliukas",
-    "email": "mantasm@wix.com"
+  "description": "React component library to generate the HTML emails on the fly",
+  "keywords": [
+    "email",
+    "mjml",
+    "react"
+  ],
+  "author": "Ian Edington <IanEdington@gmail.com>",
+  "homepage": "https://github.com/Faire/mjml-react",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Faire/mjml-react.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Faire/mjml-react/issues"
   },
-  "homepage": "https://github.com/wix-incubator/mjml-react",
-  "bugs": "https://github.com/wix-incubator/mjml-react/issues",
   "main": "dist/src/index.js",
+  "module": "dist/esm/index.js",
   "files": [
     "extensions.js",
     "utils.js",
     "dist"
   ],
+  "directories": {
+    "test": "test"
+  },
   "publishConfig": {
     "registry": "https://registry.npmjs.org/",
     "versionBumpStrategy": "patch"
   },
   "scripts": {
-    "start": "yoshi start",
-    "pretest": "yoshi build",
-    "test": "yoshi test --mocha",
-    "posttest": "yoshi lint"
-  },
-  "devDependencies": {
-    "babel-plugin-transform-runtime": "~6.23.0",
-    "chai": "~4.1.0",
-    "mjml": "^4.7.0",
-    "mjml2json": "^1.0.1",
-    "puppeteer": "^1.4.0",
-    "react": "^17.0.0",
-    "react-dom": "^17.0.0",
-    "@wix/yoshi": "^4.186.0"
+    "build": "/bin/rm -rf dist && yarn build-cjs && yarn build-esm",
+    "build-cjs": "babel --out-dir dist/src ./src",
+    "build-esm": "babel --config-file ./babel.esm.config.js --out-dir dist/esm ./src",
+    "test": "jest test"
   },
   "peerDependencies": {
     "mjml": "^4.7.0",
     "react": "^16.0.0 || ^17.0.0 || ^18.0.0",
     "react-dom": "^16.0.0 || ^17.0.0 || ^18.0.0"
   },
-  "yoshi": {
-    "hmr": "auto"
-  },
-  "module": "dist/es/src/index.js",
-  "dependencies": {
-    "@babel/runtime": "^7.14.6",
-    "babel-runtime": "~6.25.0",
-    "color": "^3.1.3",
+  "devDependencies": {
+    "@babel/cli": "^7.18.10",
+    "@babel/core": "^7.19.0",
+    "@babel/preset-env": "^7.19.0",
+    "@babel/preset-react": "^7.18.6",
+    "chai": "^4.3.6",
+    "jest": "^29.0.3",
+    "mjml": "^4.7.0",
+    "mjml2json": "^1.0.1",
+    "prop-types": "^15.8.1",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
     "react-reconciler": "^0.26.1"
```

Comment:
Is "react-reconciler" needed as a peer dependency? It looks like it used to be a dependency

Reply:
I just put back to wherever it should belong in the previous version of `package.json`